test/TestCases/loaders/i18nTypeScriptModules/index.js

```diff
@@ -0,0 +1,5 @@
+define(["dojo/i18n!./nls/strings", "./nls/de-at/strings"], function(strings) {
```

Comment:
```suggestion
define(["dojo/i18n!./nls/strings"], function(strings) {
```

Reply:
Nope, if I remove "./nls/de-at/strings" from my dependency list, it will not get the German string. Besides, this is a different problem not attacked by this test, so I would like to stay easy including this.

Reply:
Ah, I see - the patch to use the return value is helping, yes. It is very irritating, but that's how's life. :-)
Thanks a lot, @chuckdumont I am going to commit this one as well